generators/k8helmpipeline/index.js

```diff
@@ -12,6 +12,8 @@ module.exports = class extends Generator {
       // Calling the super constructor is important so our generator is correctly set up
       super(args, opts);
 
+      this.kube = true;
```

Comment:
what's this for?

Reply:
var isKube = obj.hasOwnProperty('kube'); in the target function